From johfel@gmx.de  Wed Nov 12 11:31:53 2008
From: Johann Felix Soden <johfel@gmx.de>
Date: Thu, 30 Oct 2008 22:44:39 +0100
Subject: driver core: fix using 'ret' variable in unregister_dynamic_debug_module
To: gregkh@suse.de
Cc: jbaron@redhat.com
Message-ID: <1225403079.7328.7.camel@localhost>

From: Johann Felix Soden <johfel@users.sourceforge.net>

The 'ret' variable is assigned, but not used in the return statement. Fix this.

Signed-off-by: Johann Felix Soden <johfel@users.sourceforge.net>
Acked-by: Jason Baron <jbaron@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 lib/dynamic_printk.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/dynamic_printk.c
+++ b/lib/dynamic_printk.c
@@ -135,7 +135,7 @@ int unregister_dynamic_debug_module(char
 	nr_entries--;
 out:
 	up(&debug_list_mutex);
-	return 0;
+	return ret;
 }
 EXPORT_SYMBOL_GPL(unregister_dynamic_debug_module);
 
